payload/egg/core/eggSceneManager.cc

```diff
@@ -100,11 +100,11 @@ void SceneManager::reinitCurrentScene() {
     }
 }
 
-void SceneManager::createScene(s32 sceneId, Scene *parent) {
+void SceneManager::createScene(EGG::SceneId sceneId, Scene &parent) {
```

Comment:
```suggestion
void SceneManager::createScene(EGG::SceneId sceneId, Scene *parent) {
```
The function explicitly checks if `parent` is a nullptr.